src/components/forms/SelectField/SelectField.js

```diff
@@ -0,0 +1,69 @@
+import {FormFieldDescription, FormLabel, Paragraph} from '@utrecht/component-library-react';
+import {Field} from 'formik';
+import PropTypes from 'prop-types';
+import Select from 'react-select';
+
+import {getBEMClassName} from '../../../utils';
+
+export const SelectField = ({
+  name = '',
```

Comment:
```suggestion
  name,
```
it's a required prop (due to how we use formik), so no point in providing a default.